apps/cowswap-frontend/src/modules/limitOrders/pure/DeadlineSelector/deadlines.ts

```diff
@@ -11,35 +11,18 @@ export interface LimitOrderDeadline {
 export const MIN_CUSTOM_DEADLINE = ms`30min`
 export const MAX_CUSTOM_DEADLINE = MAX_ORDER_DEADLINE
 
-export enum LimitOrderDeadlinePreset {
-  FIVE_MINUTES = '5 Minutes',
-  THIRTY_MINUTES = '30 Minutes',
-  ONE_HOUR = '1 Hour',
-  ONE_DAY = '1 Day',
-  THREE_DAYS = '3 Days',
-  ONE_MONTH = '1 Month',
-  ONE_YEAR = '1 Year (max)',
-}
-
-const DEADLINE_VALUES: Record<LimitOrderDeadlinePreset, number> = {
-  [LimitOrderDeadlinePreset.FIVE_MINUTES]: ms`5m`,
-  [LimitOrderDeadlinePreset.THIRTY_MINUTES]: ms`30m`,
-  [LimitOrderDeadlinePreset.ONE_HOUR]: ms`1 hour`,
-  [LimitOrderDeadlinePreset.ONE_DAY]: ms`1d`,
-  [LimitOrderDeadlinePreset.THREE_DAYS]: ms`3d`,
-  [LimitOrderDeadlinePreset.ONE_MONTH]: ms`30d`,
-  [LimitOrderDeadlinePreset.ONE_YEAR]: MAX_CUSTOM_DEADLINE,
-}
-
-export const defaultLimitOrderDeadline: LimitOrderDeadline = {
-  title: LimitOrderDeadlinePreset.ONE_YEAR,
-  value: DEADLINE_VALUES[LimitOrderDeadlinePreset.ONE_YEAR],
-}
-
-export const LIMIT_ORDERS_DEADLINES: LimitOrderDeadline[] = Object.entries(DEADLINE_VALUES).map(([title, value]) => ({
-  title,
-  value,
-}))
+export const defaultLimitOrderDeadline: LimitOrderDeadline = { title: '7 Days', value: ms`7d` }
+
+export const LIMIT_ORDERS_DEADLINES: LimitOrderDeadline[] = [
+  { title: '5 Minutes', value: ms`5m` },
+  { title: '30 Minutes', value: ms`30m` },
+  { title: '1 Hour', value: ms`1 hour` },
+  { title: '1 Day', value: ms`1d` },
+  { title: '3 Days', value: ms`3d` },
+  defaultLimitOrderDeadline,
+  { title: '1 Month', value: ms`30d` },
+  { title: '1 Year (max)', value: MAX_CUSTOM_DEADLINE },
```

Comment:
Great refactor also. 

nit: I didn't get why we redeclare `MAX_ORDER_DEADLINE` under a different name `MAX_CUSTOM_DEADLINE`

Also, to avoid the value of `MAX_ORDER_DEADLINE` and the title (`1 Year (max)`), I think we should have both definition next to each other.
```ts
  { title: `${MAX_CUSTOM_DEADLINE.label} (max)`, value: MAX_CUSTOM_DEADLINE.value },
```